recipes/wayland-protocols/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class WaylandProtocolsConan(ConanFile):
+    name = "wayland-protocols"
+    description = "Wayland is a project to define a protocol for a compositor to talk to its clients as well as a library implementation of the protocol"
+    topics = ("wayland")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.freedesktop.org/wayland/wayland-protocols"
+    license = "MIT"
+
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Do we really need these settings for generating xml files ?

Reply:
Some of them are required by Meson. Anyway, it is declared as `header_only` so only one package is generated.

Reply:
Lgtm